doc/quick-start-guide.md

```diff
@@ -185,6 +185,27 @@ First, we see some information about the host system (the machine that the toolk
 
 When you run into problems with your toolkit, you should first run the doctor script and check it's output. 
 
+## LaTeX environment
+
+To save bandwidth, the Overleaf image only comes with a minimal install of TeXLive. To upgrade to a complete TeXLive installation, run the installation script in the Overleaf container with the following command
+
+```sh
+$ docker exec sharelatex tlmgr install scheme-full
```

Comment:
This is not sufficient anymore. `tlmgr path add` is required for linking all the bin/ scripts.